provision.yaml

```diff
@@ -19,4 +19,4 @@ travis_el7:
   images: ['litmusimage/centos:7', 'litmusimage/oraclelinux:7', 'litmusimage/scientificlinux:7']
 release_checks:
   provisioner: vmpooler
-  images: ['redhat-5-x86_64', 'redhat-7-x86_64', 'centos-5-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'centos-8-x86_64', 'oracle-5-x86_64', 'oracle-6-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64', 'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64' ]
+  images: [ 'redhat-7-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'centos-8-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64', 'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64' ]
```

Comment:
I think we should add `oracle-7-x86_64` to the tests, given we're [still supporting it].

Reply:
HI @sanfrancrisko,
A new ticket(IAC-775) was created for adding oracle-7-x86_64 to release_checks.